src/main/java/com/ifactory/press/db/solr/processor/UpdateDocValuesProcessor.java

```diff
@@ -127,10 +127,10 @@ private void retrieveDocValuesHelper(String[] valueFields,
     Term idTerm = new Term(idField, id);
     TermQuery query = new TermQuery(idTerm);
     // Assert TermState is not used and that objects are not null
-    TopDocs docs = query.getTermContext() == null ? searcher.search(query, 1) : null;
+    TopDocs docs = searcher.search(query, 1) ;
```

Comment:
unsure if this change is save but getTermContext() no longer exists

Reply:
Yes this one is deprecated, I changed it in branch. It changed to `.getTermStates()`